zipkin-storage/influxdb/src/main/java/zipkin2/storage/influxdb/InfluxDBSpanStore.java

```diff
@@ -255,10 +255,15 @@ private String trimLeadingZeroes(String str){
     }
 
     List<String> services = new ArrayList<>();
-    for (QueryResult.Result result: response.getResults()){
-      for (QueryResult.Series series : result.getSeries()){
-        for (List<Object> values: series.getValues()) {
-          services.add(values.get(1).toString());
+    List<QueryResult.Result> results = response.getResults();
+    if (results != null) {
+      for (QueryResult.Result result : results) {
+        if (result != null && result.getSeries() != null) {
+          for (QueryResult.Series series : result.getSeries()) {
+            for (List<Object> values : series.getValues()) {
+              services.add(values.get(1).toString());
```

Comment:
This may cause a NPE if there is a null value returned by InfluxDB. Would be possible to use `String.valueOf(...)`?